diff --git a/internal/pkg/envcheck/docker.go b/internal/pkg/envcheck/docker.go
index 0a634ad..173b53a 100644
--- a/internal/pkg/envcheck/docker.go
+++ b/internal/pkg/envcheck/docker.go
@@ -5,30 +5,30 @@ package envcheck
 
 import (
 	"context"
-	"fmt"
+	// "fmt"
 
-	"github.com/docker/docker/client"
+	// "github.com/docker/docker/client"
 )
 
 func CheckIfDockerRunning(ctx context.Context, configDockerHost string) error {
-	opts := []client.Opt{
-		client.FromEnv,
-	}
-
-	if configDockerHost != "" {
-		opts = append(opts, client.WithHost(configDockerHost))
-	}
-
-	cli, err := client.NewClientWithOpts(opts...)
-	if err != nil {
-		return err
-	}
-	defer cli.Close()
-
-	_, err = cli.Ping(ctx)
-	if err != nil {
-		return fmt.Errorf("cannot ping the docker daemon, does it running? %w", err)
-	}
+	// opts := []client.Opt{
+	// 	client.FromEnv,
+	// }
+
+	// if configDockerHost != "" {
+	// 	opts = append(opts, client.WithHost(configDockerHost))
+	// }
+
+	// cli, err := client.NewClientWithOpts(opts...)
+	// if err != nil {
+	// 	return err
+	// }
+	// defer cli.Close()
+
+	// _, err = cli.Ping(ctx)
+	// if err != nil {
+	// 	return fmt.Errorf("cannot ping the docker daemon, does it running? %w", err)
+	// }
 
 	return nil
 }
